docs/reference/hooks.rst

```diff
@@ -19,17 +19,17 @@ The DbtRemoteHook interface
 *dbt* git remote
 ^^^^^^^^^^^^^^^^
 
-.. automodule:: airflow_dbt_python.hooks.git
+.. automodule:: airflow_dbt_python.hooks.remote.git
    :members:
 
 *dbt* localfs remote
 ^^^^^^^^^^^^^^^^^^^^
 
-.. automodule:: airflow_dbt_python.hooks.localfs
+.. automodule:: airflow_dbt_python.hooks.remote.localfs
```

Comment:
It is very contradictory to read `remote.local`, maybe we could rename this one to just `fs`?

Reply:
I'm in favor of renaming it to `fs`